imsv-docs-docusaurus/openapi/endpoints/kyc/submit-partner-kyc-statement.yaml

```diff
@@ -17,6 +17,60 @@ requestBody:
     application/json:
       schema:
         $ref: './models/submit-kyc-statement-request.yaml'
+      examples:
+        NZ Passport:
+          summary: NZL Passport for Person in New Zealand
+          value:
+            {
+              "region": "NZL",
+              "idempotencyKey": "21ede8ef0d230d0a6bf755e07d39la54",
+              "claims": [
+                {
+                  "claimType": "FULL_NAME",
+                  "attributes": {
+                    "givenName": "John",
+                    "middleName": "Foo Bar",
+                    "familyName": "Doe",
+                    "honorific": "Mr"
+                  }
+                }
+              ],
+              "evidence": [
+                {
+                  "evidenceType": "PASSPORT",
+                  "documentId": "DS230475",
+                  "country": "NZL",
+                  "expiry": "2024-10-12"
+                },
+              ]
+            }
+        Australian Drivers License:
+          summary: Australian Drivers License issued by State of Victoria for Person in Australia
+          value:
+            {
+              "idempotencyKey": "21ede8ef0d230d0a6bf755e07d39la54",
+              "region": "AUS",
+              "claims": [
+                {
+                  "claimType": "FULL_NAME",
+                  "attributes": {
+                    "givenName": "John",
+                    "middleName": "Foo Bar",
+                    "familyName": "Doe",
+                    "honorific": "Mr"
+                  }
+                }
+              ],
+              "evidence": [
+                {
+                  "evidenceType": "DRIVERS_LICENSE",
+                  "documentId": "DS230475",
+                  "country": "AUS",
+                  "region": "VIC",
+                  "version": "978"
```

Comment:
HZRHQO0GS8 would look more plausible as an example of an AU license (taken from here https://airtable.com/app379qZK2sJMB3pu/shrrvIFezuGcuVNvW/tblzcpgpWahoNbHV5)